layersvt/device_simulation.cpp

```diff
@@ -1482,6 +1482,35 @@ VKAPI_ATTR void VKAPI_CALL GetPhysicalDeviceFormatProperties2KHR(VkPhysicalDevic
     GetPhysicalDeviceFormatProperties(physicalDevice, format, &pFormatProperties->formatProperties);
 }
 
+VKAPI_ATTR VkResult VKAPI_CALL GetPhysicalDeviceToolPropertiesEXT(VkPhysicalDevice physicalDevice, uint32_t *pToolCount,
+                                                                  VkPhysicalDeviceToolPropertiesEXT *pToolProperties) {
+    static const VkPhysicalDeviceToolPropertiesEXT devsim_layer_tool_props = {VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_TOOL_PROPERTIES_EXT,
+                                                                              nullptr,
+                                                                              "Device Simulation Layer",
+                                                                              "1",
```

Comment:
Should the tool version be the layer's version?

Reply:
See above answer.